TypeCobol/Tools/APIHelpers/Helpers.cs

```diff
@@ -73,11 +73,42 @@ public static SymbolTable LoadIntrinsic(List<string> paths, DocumentFormat intri
             return table;
         }
 
-        public static SymbolTable LoadDependencies([NotNull] List<string> paths, DocumentFormat format, SymbolTable intrinsicTable,
-            [NotNull] List<string> inputFiles, List<string> copyFolders, EventHandler<DiagnosticsErrorEvent> diagEvent)
+        private static CompilationUnit ParseDependency(string path, DocumentFormat format, SymbolTable intrinsicTable,
+            List<string> copyFolders)
         {
-
             var parser = new Parser(intrinsicTable);
+
+            parser.Init(path, new TypeCobolOptions { ExecToStep = ExecutionStep.SemanticCheck }, format, copyFolders);
+            parser.Parse(path); //Parse the dependency file
+
+            return parser.Results;
+        }
+
+        public static IEnumerable<string> GetDependenciesMissingCopies([NotNull] List<string> paths, DocumentFormat format, SymbolTable intrinsicTable, List<string> copyFolders, EventHandler<DiagnosticsErrorEvent> diagEvent)
```

Comment:
The return type could be a `IDictionary<string, IList<string>>` to give the list of missing copies for each dependency file.

Reply:
The result of this method is not meant to be displayed to the user, only to download the missing copies. It is useless to return the list of missing copies per file here.
However, a MissingCopyException writes a file that misses some copies. MissingCopyExceptions can be thrown after a LoadDependencies, it can be interesting to return such a list there.

Reply:
MissingCopyException is only used in CLI and it already got the filePath for which copies are missing.